app/assets/javascripts/components/common/AssignCell/AssignButton.jsx

```diff
@@ -243,14 +243,24 @@ export class AssignButton extends React.Component {
     e.preventDefault();
 
     const title = e.target.value;
-    const assignment = CourseUtils.articleFromTitleInput(title);
-    const language = assignment.language || this.state.language;
-    const project = assignment.project || this.state.project;
+    const articlesTitles = [];
+    let language;
+    let project;
+    title.split('\n').forEach((articleTitle) => {
+      if (!articleTitle) {
+        articlesTitles.push('');
```

Comment:
What's this for?

Reply:
the pushing of the empty string? That's to make sure that user entered new lines are preserved - without it, pressing enter would do essentially nothing.

Reply:
Maybe add a comment to that effect.